README.md

```diff
@@ -6,7 +6,7 @@ Jenkins Remoting layer
 
 Jenkins remoting is an executable JAR, 
 which implements communication layer in [Jenkins](https://jenkins.io) automation server. 
-It's being used for master <=> agent(fka "slave") and master <=> CLI communications.
```

Comment:
I think it still makes sense to mention it here. We can amend it somehow to be explicit that the term is discouraged

Reply:
I thought about it, but I don't think we need to maintain this mention here. We could add it as a footnote somewhere. I've thought about trying to clean up the documentation some. I could add a note in there somewhere.

It's been quite a while since the terminology change and there are no shortage of places that mention that agents used to be known as slaves. Probably don't need to keep this one.